guides/common/modules/proc_creating-a-custom-ssl-certificate.adoc

```diff
@@ -60,6 +60,22 @@ authorityKeyIdentifier=keyid,issuer
 [ alt_names ]
 DNS.1 = _{context}.example.com_
 ----
+Optional: If you want to add Distinguished Name (DN) details to the CSR, add the following information to the `[ req_distinguished_name ]` section:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+[req_distinguished_name]
+CN = _{context}.example.com_
+countryName =_My_Country_Name_ <1>
+stateOrProvinceName = _My_State_Or_Province_Name_ <2>
+localityName = _My_Locality_Name_ <3>
+organizationName = _My_Organization_Or_Company_Name_
+organizationalUnitName = _My_Organizational_Unit_Name_ <4>
+----
+<1> Country Name (2 letter code)
+<2> State or Province Name (full name)
+<3> Locality Name (example: New York)
+<4> Organizational Unit Name (example: IT)
```

Comment:
I think the stuff you have here outside of parentheses is redundant, because it is already in the code block. What do you think @apinnick?

Reply:
Not sure. If there are constraints (2-letter country code, for example), we need to call them out. If the other fields are self-evident, then they probably do not need callouts. I imagine that state/province would not need to be documented. 

My only question is whether all these fields are required. Would it be possible to add a DN without a state/province, for example? If a user lives in Vatican City, that would be relevant. OTOH, if the fields are not validated in any way, then we would not need detailed documentation.

Reply:
Constraints are perfectly fine. What I meant was not to repeat the actual name of the field like this:
```suggestion
<1> 2 letter code.
<2> Full name.
<3> For example, New York
<4> For example, IT department
```

Reply:
@apinnick some of these fields are optional, however, we have a bug asking that this information be expressly mentioned in the CSR section to increase trust.

Reply:
@apinnick I have added "Optional" at the beginning of the preceding sentence. Does that work?

Reply:
LGTM